centreon/src/Core/ResourceAccess/Application/UseCase/DeleteRule/DeleteRule.php

```diff
@@ -129,24 +126,13 @@ private function isAuthorized(): bool
     private function deleteRule(int $ruleId): void
     {
         $this->debug('Starting transaction');
-        $this->dataStorageEngine->startTransaction();
-
-        try {
-            /**
-             * Here are the deletions (on cascade or not) that will occur on rule deletion
-             *     - Contact relations (ON DELETE CASCADE)
-             *     - Contact Group relations (ON DELETE CASCADE)
-             *     - Datasets relations + datasets (NEED MANUAL DELETION)
-             *     - DatasetFilters (ON DELETE CASCADE).
-             */
-            $this->writeRepository->deleteRuleAndDatasets($ruleId);
-            $this->debug('Commit transaction');
-            $this->dataStorageEngine->commitTransaction();
-        } catch (\Throwable $exception) {
-            $this->debug('Rollback transaction');
-            $this->dataStorageEngine->rollbackTransaction();
-
-            throw $exception;
-        }
+        /**
+         * Here are the deletions (on cascade or not) that will occur on rule deletion
+         *     - Contact relations (ON DELETE CASCADE)
+         *     - Contact Group relations (ON DELETE CASCADE)
+         *     - Datasets relations + datasets (NEED MANUAL DELETION)
+         *     - DatasetFilters (ON DELETE CASCADE).
+         */
```

Comment:
This information should not be included here, as it is technical information that must be included in the repository